iib/workers/config.py

```diff
@@ -30,6 +30,12 @@ class Config(object):
     )
     iib_request_logs_level = 'DEBUG'
     iib_required_labels = {}
+    # Configuration for dogpile.cache
+    # Disabled by default (by using 'dogpile.cache.null').
+    # To enable caching set 'dogpile.cache.memcached' as backend.
+    iib_dogpile_backend = 'dogpile.cache.null'
+    iib_dogpile_expiration_time = 5
```

Comment:
This seems like a very short default and would not last the whole request. Do you want to set this to something like 10 minutes by default?

Reply:
Oh noes, this one was committed by mistake. 
Thanks!